File: Utility/CustomCommands.vb
Imports System.Windows.Input

Public Class CustomCommands
    Public Shared About As New RoutedUICommand("About",
                                               "About",
                                               GetType(CustomCommands))

    Public Shared Close As New RoutedUICommand("Close",
                                               "Close",
                                               GetType(CustomCommands))

    Public Shared Generate As New RoutedUICommand("Generate",
                                                  "Generate",
                                                  GetType(CustomCommands))

    Public Shared SelectNone As New RoutedUICommand("Select None",
                                                    "SelectNone",
                                                    GetType(CustomCommands))

    Public Shared Cancel As New RoutedUICommand("Cancel",
                                                "Cancel",
                                                GetType(CustomCommands))
    Public Shared Add As New RoutedUICommand("Add",
                                             "Add",
                                             GetType(CustomCommands))
    Public Shared Delete As New RoutedUICommand("Delete",
                                                "Delete",
                                                GetType(CustomCommands))
End Class

File: Utility/CustomCommands.cs
using System.Windows.Input;

namespace Utility
{
    public class CustomCommands
    {
        public static RoutedUICommand About = new RoutedUICommand("About", "About", typeof (CustomCommands));
        public static RoutedUICommand Close = new RoutedUICommand("Close", "Close", typeof (CustomCommands));
        public static RoutedUICommand Generate = new RoutedUICommand("Generate", "Generate", typeof (CustomCommands));
        public static RoutedUICommand SelectNone = new RoutedUICommand("Select None", "SelectNone", typeof (CustomCommands));
        public static RoutedUICommand Cancel = new RoutedUICommand("Cancel", "Cancel", typeof (CustomCommands));
        public static RoutedUICommand Add = new RoutedUICommand("Add", "Add", typeof (CustomCommands));
        public static RoutedUICommand Delete = new RoutedUICommand("Delete", "Delete", typeof (CustomCommands));
    }
}